client/src/app/services/chart.service.ts

```diff
@@ -525,13 +540,60 @@ export class ChartService {
   }
   //#endregion
 
+  //#region WINDOW AND TIMEFRAME OPERATIONS
```

Comment:
remove obsolete “AND TIMEFRAME” from code comment and mention of it in PR description, examples